src/Job/Message.php

```diff
@@ -95,19 +93,17 @@ public function getCallable()
         if ($this->callable) {
             return $this->callable;
         }
+
         $target = $this->parsedBody['class'] ?? null;
 
-        $callable = null;
-        if (is_array($target) && count($target) === 2) {
-            $instance = new $target[0]();
-            $callable = Closure::fromCallable([$instance, $target[1]]);
-        } elseif (is_string($target)) {
-            /** @psalm-suppress InvalidArgument */
-            $callable = Closure::fromCallable($target);
```

Comment:
This form was so you could attach global functions as 'jobs'. I'm ok with this being a breaking change as it isn't a great pattern to encourage folks to use when the Job class is simple to implement.